src/components/BannerIntroduction.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div class="s-banner-introduction" :style="{ 'background-color': bgColor }">
+    <slot name="bgImage" />
+    <div class="s-banner-introduction__body">
+      <h2 class="s-banner-introduction__title">
+        <slot name="title" />
+      </h2>
+      <p class="s-banner-introduction__description">
+        <slot name="description" />
+      </p>
+    </div>
+    <div class="s-banner-introduction__button">
+      <slot name="button" />
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from "vue-property-decorator";
+
+@Component({})
+export default class BannerIntroduction extends Vue {
+  @Prop(String)
+  bgColor!: string;
+}
+</script>
+
+<style lang="less">
+@import (reference) "./../styles/Imports";
+
+.s-banner-introduction {
+  box-shadow: 0px 2px 8px rgba(9, 22, 29, 0.16);
+  border-radius: 8px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 12px 16px;
+  position: relative;
+
+  &__title,
+  &__description {
+    margin: 0;
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+    line-height: 130%;
+    color: @white !important;
```

Comment:
There a reason why you used `!important` here?

Reply:
https://gyazo.com/68b38f8c46a1313b059e764a9d2ebdb5 This style is blocking now

Reply:
Sorry, fixed it
`.night {
  .s-banner-introduction {
    &__description {
      color: @white;
    }
  }
}`